src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  margin: unset;
```

Comment:
`unset` sets the value to inherited or initial value, to **reset** the default margin, you need to just do `margin: 0`